service-registry-cli/README.adoc

```diff
@@ -0,0 +1,327 @@
+:product-long: Application Services
+:product: App Services
+:registry-product-long: OpenShift Service Registry
+:registry: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://console.redhat.com/beta/application-services/streams/
+:registry-url: https://console.redhat.com/beta/application-services/service-registry/
+:property-file-name: app-services.properties
+:rhoas-version: 0.29.0
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+:consumer-config-url: consumer-configuration/README.adoc
+:service-binding-url: service-discovery/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-getting-started-rhoas-cli-registry"]
+= Getting started with the Service Registry using rhoas CLI
+:context: getting-started-rhoas-registry
+
+[IMPORTANT]
+====
+The `rhoas` command-line interface (CLI) is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+[role="_abstract"]
+With the `rhoas` command-line interface (CLI), you can manage your application services from a terminal.
+This guide describes how to get started quickly with Service Registry and the RHOAS CLI.
+
+--
+* Using `rhoas` CLI to manage Service Registry instances .
+
+* Retrieve configuration to use Service Registry instance in your application
+
+* Manage Service Registry Artifacts directly from command line
+--
+
+[id="proc-installing-rhoas_{context}"]
+== Installing the rhoas CLI
+
+[role="_abstract"]
+If you haven't installed RHOAS CLI you can install it using the following guide
+ TODO link to CLI installation guide
+
+NOTE:  Service Registry integration requires *latest* version of RHOAS CLI
+
+
+[id="proc-using-rhoas_registry{context}"]
+== Using rhoas CLI to manage Service Registry instances
+
+[role="_abstract"]
+As a developer of applications and services,
+you can use `rhoas` to create Service Registry instances and connect your applications and services to these instances.
+
+The following procedures demonstrate a basic workflow to get you started:
+
+> TODO - correct urls
```

Comment:
I'm missing info how rhoas-cli-url is generated. Was inspired by other CLI guide to list operations we do